testapp/src/main/AndroidManifest.xml

```diff
@@ -5,8 +5,8 @@
     android:versionName="1.0" >
 
     <uses-sdk
-        android:minSdkVersion="9"
-        android:targetSdkVersion="19" />
+        android:minSdkVersion="14"
+        android:targetSdkVersion="22"/>
```

Comment:
Please always set the `targetSdkVersion` to the current stable release of Android, API 25.